crates/gpui/src/platform/windows/util.rs

```diff
@@ -100,3 +104,34 @@ pub(crate) fn create_event() -> windows::core::Result<OwnedHandle> {
 pub(crate) fn windows_credentials_target_name(url: &str) -> String {
     format!("zed:url={}", url)
 }
+
+pub(crate) fn load_cursor(style: CursorStyle) -> HCURSOR {
+    static ARROW: OnceLock<HCURSOR> = OnceLock::new();
+    static IBEAM: OnceLock<HCURSOR> = OnceLock::new();
+    static CROSS: OnceLock<HCURSOR> = OnceLock::new();
+    static HAND: OnceLock<HCURSOR> = OnceLock::new();
+    static SIZEWE: OnceLock<HCURSOR> = OnceLock::new();
+    static SIZENS: OnceLock<HCURSOR> = OnceLock::new();
```

Comment:
Is this really needed? As we call `LoadImageW` with `LR_SHARED`, windows will cache images.

Reply:
Although I did not measure it strictly, there is a speed difference of about 1,000 times, because there are no jumps to non-optimizable external libraries or system calls in between.
However, this is not a performance-critical part, so it may not be necessary.

Reply:
I haven't been involved in real-world projects before. How do you guys typically measure the performance of these functions in your read-world work? Is it done through tools like `Tracy` or simply using basic methods like `std::time`?

Reply:
BTW, keep the changes plz, as the 1000 times perf diff.

Reply:
https://github.com/kazatsuyu/zed/commit/1274d097039c727796922fb5709d26299f031b39
I just tried to loop it in a rough way, so I redid it using `#[bench]` properly. Note that it only works in the nightly toolchain.

The result is as follows:

```
test platform::windows::util::tests::bench_load_cursor   ... bench:           5 ns/iter (+/- 0)
test platform::windows::util::tests::bench_load_cursor_2 ... bench:       4,262 ns/iter (+/- 1,584)
```